### PR TITLE
Add VPC ID to outputs and is_default variable

### PR DESCRIPTION
**Pull request by dkossako**
This commit adds VPC ID to outputs of terraform. ID is required to pass as an input to kubernetes module. `is_default` variable allows to create default projects.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -25,5 +25,6 @@ variable "project" {
     description = "My project"
     purpose     = "Personal"
     environment = "development"
+    is_default  = "false"
```
Comment (brucellino) [inline variables.tf:28]: This is a boolean - could you please update the map with the correct signature to include it?

Furthermore, the default project should be the default, so I would set the default of `is_default` to `true`.
Thanks!

**Review by brucellino — CHANGES_REQUESTED** [review]
Hello! Thanks very much for the PR - I agree it makes sense to set whether the project created is the default.

Could you perhaps make the minor modifications I suggested in the review?
Thanks!
Bruce

File: outputs.tf
Comment (brucellino) [inline outputs.tf:1]: LGTM

**Comment by dkossako** [comment]
Updated

**Review by brucellino — APPROVED** [review]
much obliged!

**Comment by dkossako** [comment]
There's something wrong with test, but it looks like some token is missing? Or something else?

**Comment by brucellino** [comment]
Fixed tests, but can't re-run the check. Tested locally. Merging.

**Pull request merged**